### ISSUE TITLE
Output of the Rich Text Editor does not match with the input.  

### ISSUE THREAD

**Issue by govindvish** [tinymce/tinymce-react#81]
Hi
I am creating a ticket page where the description box is a Tinymce Rich Text Editor.
When I'm inserting data in the form of HTML Tags, I'm able to view the result in the Editor on Ticket Create Page as:

![Screenshot1](https://user-images.githubusercontent.com/51371329/61630155-a6872700-aca4-11e9-95e8-4041f23257a8.png)

But when I'm fetching the data for the description box from DataBase on Ticket View Page, I want the Editor to show result same as it was while inserting data i.e as in Screenshot1,
Instead I'm getting the following as output in the Editor on the Ticket View Page:

![Screenshot2](https://user-images.githubusercontent.com/51371329/61630668-f1ee0500-aca5-11e9-8329-06014b196da2.png)

**Comment by SimonFc** [comment]
Hi!
How are you setting the content of the editor after you've fetched it? If you're using `setContent` you should be able to specify the format like this:
```editor.setContent('<p>Some Content</p>', { format: 'html' });```
Does that solve your issue?

**Comment by govindvish** [comment]
Hi
I've done the following in my code.
Ticket Create Page Code Snippet:
`<Editor id={'ID'} value={this.state.content} onEditorChange={this.handleEditorChange} />`

Ticket View Page Code Snippet:
`const editor = tinymce && tinymce.get('ID')`
`return (<Editor disabled='true' init={{ setup: (editor) => this.editor = editor, toolbar: false }} value= 
   {editor.setContent(props.dataTicket.content, { format: 'html' })} /> )`

Please Help me out As I'm new to React.!

**Comment by SimonFc** [comment]
Ah okay I think I understand your use case. So you have two editors, one where you do the editing and disabled which you use as output, right?

Ignore the thing I said earlier with setContent. You should be able to do it as a controlled component:
```
<Editor onEditorChange={this.handleEditorChange} />
```
Where handleEditorChange sets this.state.content, and then in the second editor:
```
<Editor
    value={this.state.content}
/>
```

**Comment by govindvish** [comment]
Hi
I've done what you mentioned above.
Snapshot of my Ticket Create Page:
![CreateTicket](https://user-images.githubusercontent.com/51371329/61700053-a8f78880-ad59-11e9-9903-8b636e35b7b1.png)

Snapshot of my Ticket View Page:
![TicketView](https://user-images.githubusercontent.com/51371329/61700389-40f57200-ad5a-11e9-9bb6-d3dbf5bbd14d.png)

But still I'm getting the same issue as mentioned in my first comment i.e input and output are different.

**Comment by SimonFc** [comment]
What's the exact value that's producing the incorrect output? See if you can reproduce the issue over here: https://codesandbox.io/embed/tinymcereact-m7eur?fontsize=14

**Comment by govindvish** [comment]
Hi.
I found out the issue and it was nothing related to your package.
The issue with me was that when I fetched data from my database it was in the form of 
`&lt;p&gt;Hello World&lt;/p&gt;`
So I needed to parse my data in order to get desired result, which I did it with the help of another package i.e [html-react-parser](https://github.com/remarkablemark/html-react-parser).
And then I passed my parsed data to the 'value' prop of the Editor component of your package.
And this is how I resolved my issue.
**Thanks a lot for all the support.**
